ios/engine/KMEI/KeymanEngine/Classes/kmp.json/KMPInfo.swift

```diff
@@ -19,13 +19,16 @@ class KMPInfoItem: Codable {
 }
 
 class KMPInfo: Codable {
+  static let DEFAULT_VERSION = KMPInfoItem(description: "1.0.0")
+  static let AUTOGEN_VERSION = KMPInfoItem(description: "0.0.0")
+
   var name: KMPInfoItem? = nil
   var author: KMPInfoItem? = nil
   var copyright: KMPInfoItem? = nil
   var website: KMPInfoItem? = nil
   // If we're constructing one from scratch, set to the absolute minimal version.
   // This will automatically signal that the 'package' needs to be updated.
-  var version: KMPInfoItem? = KMPInfoItem(description: "0.0.0")
+  var version: KMPInfoItem? = KMPInfo.DEFAULT_VERSION
```

Comment:
This is a little different -- it's now giving `1.0.0` which can be higher than some package versions, as the default version, so at the very least the comment on the line above doesn't fit.

Reply:
Right, it belongs elsewhere now.  I'll get that patched up shortly.